Readme.md

```diff
@@ -21,9 +21,16 @@ To build and run this example the following requirements are necessary:
 
 TX and RX pin definitions are target specific - the code and the physical board must both be using the same pins:
 
-1. To know which pins the code is looking for, see the ``serial_asynch.cpp`` file for this sample. The pins are defined as ``TEST_SERIAL_ONE_TX_PIN`` and ``TEST_SERIAL_TWO_RX_PIN``.
+1. To know which pins the code is looking for, see the ``serial_asynch.cpp`` file for this sample. The pins are defined as ``YOTTA_CFG_HARDWARE_TEST_PINS_SERIAL_TX`` and ``YOTTA_CFG_HARDWARE_TEST_PINS_SERIAL_RX``.
```

Comment:
We could removed 1. , as pins were defined in the code file, they are now in the config, which is already explained in the 2nd. Shall we merge 1 and 2?